lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -169,9 +169,7 @@ function dosomething_reportback_files_form($form, &$form_state, $status = NULL,
 
   foreach ($result as $record) {
     $rb_path = 'admin/reportback/' . $record->rbid;
-    // @TODO: temporarily disable pre-approved to unblock campaigns team from reviewing reportbacks.
-    // Refs #4797
-    // $status = 'approved';
+    $status = 'approved';
```

Comment:
The illegal choice error was that we previously were **requiring** a status to be selected, and since we turned off selecting "approved" it was throwing the error for the recently submitted RB that it was bringing into the query of items to consider.